spec/services/case_extend_for_pit_service_spec.rb

```diff
@@ -24,9 +24,9 @@
       service.call
       expect(case_being_drafted.state_machine)
         .to have_received(:extend_for_pit!)
-              .with manager,
-                    10.business_days.after(old_external_deadline),
-                    'I like to extend my best tests'
+              # .with manager,
```

Comment:
Is this meant to be commented out?

Reply:
Nope - the method signature has changed.  Will fix.